### PR TITLE
Added Projects API

### PR DESCRIPTION
**Pull request by ghost**
We are missing update projects endpoint, but we have a test for it that is failing and waiting for them to fix: cognitedata/service-contracts#227

### CODE REVIEW

File: src/standardMethods.ts [hunk 1]
```diff
@@ -280,3 +280,23 @@ export function generateInsertEndpoint<RequestParams>(
...
   };
 }
+
+/** @hidden */
+export function generateSingleReplaceEndpoint<RequestType, RepsonseType>(
+  axiosInstance: AxiosInstance,
+  resourcePath: string,
+  metadataMap: MetadataMap
+) {
+  return async function replace(change: RequestType): Promise<RepsonseType> {
```
Comment (polomani) [inline src/standardMethods.ts]: maybe better name for `change`, like `replacement`?

File: src/types/types.ts [hunk 2]
```diff
@@ -793,6 +793,110 @@ export interface ServiceAccountInput {
...
+export type ProjectName = string;
+
+/**
+ * The url name of the project. This is used as part of API calls. It should only contain letters, digits and hyphens, as long as the hyphens are not at the start or end.
+ * @example publicdata
+ */
+export type UrlName = string;
+
+/**
+ * A default group for all project users. Can be used to establish default capabilities.WARNING: this group may be logically deleted
```
Comment (polomani) [inline src/types/types.ts]: space before warning

File: src/standardMethods.ts (hunk 1 above)
Comment (ghost) [reply src/standardMethods.ts]: Fixed

File: src/types/types.ts (hunk 2 above)
Comment (ghost) [reply src/types/types.ts]: Fixed

**Review by polomani — APPROVED** [review]

**Pull request merged**